checker/jdk/index/src/java/io/PipedInputStream.java

```diff
@@ -155,7 +155,7 @@ public PipedInputStream(@NonNegative int pipeSize) {
         initPipe(pipeSize);
     }
 
-    private void initPipe(@NonNegative int pipeSize) {
+    private void initPipe(@Positive int pipeSize) {
```

Comment:
Since this method is private to the JDK, we should probably just remove the annotation from it altogether.

Reply:
I agree private methods don't *need* annotations.
However, I see no harm in writing them if they are accurate.
Eventually, we would like to type-check the JDK rather than trusting the annotations, and at that time the annotations will come in useful.